experimental_setup.yaml

```diff
@@ -105,3 +105,5 @@ recourse_methods:
     hyperparams:
       loss_type: "BCE"
       binary_cat_features: False
+  geco:
+    hyperparams:
```

Comment:
We don't have any hyper parameters here to configure?

Reply:
Added some relevant ones